### ISSUE TITLE
make sure that our T1Overflow works with new JobRouter

### ISSUE THREAD

**Issue by belforte** [dmwm/CRABServer#8577]
HTCondor JobRouter will remove some deprecated functionalities in HTC 24.0
see https://htcondor.readthedocs.io/en/main/grid-computing/job-router.html
Those will also disappear in some 23.x but at least Stefano does not know `x` currently.

(those deprecations notices are there since at least 3 years)

- [x] review our use of JobRouter and check if we are using them
- [ ] if yes, do the needed changes and validate before we can switch production schedulers to new HTC

**Comment by belforte** [comment]
rats
![image](https://github.com/user-attachments/assets/664cd184-a874-4fe1-b02f-39f151779afd)

and of course our [config](https://gitlab.cern.ch/ai/it-puppet-hostgroup-vocmsglidein/-/blob/master/code/templates/modules/condor/90_jobrouter.config.erb#L11) is based on `JOB_ROUTER_ENTRIES_CMD`

which means that we need to fully review/redo

**Comment by belforte** [comment]
At first sight we can replace the script with a set of static route tables, one for each T1.

**Comment by belforte** [comment]
On HOLD waiting for #8660

**Comment by aspiringmind-code** [comment]
@belforte Since #8660 is completed, should we remove the On Hold status?

**Comment by aspiringmind-code** [comment]
Fixed in https://gitlab.cern.ch/ai/it-puppet-hostgroup-vocmsglidein/-/merge_requests/260
